src/vs/workbench/parts/terminal/electron-browser/terminalProcess.js

```diff
@@ -48,6 +48,7 @@ ptyProcess.on('data', function (data) {
 });
 
 ptyProcess.on('exit', function (exitCode) {
+	ptyProcess.kill();
```

Comment:
@chirag64 well it was just a suspicious that we may need a timeout, if it works fine without it that's fine.